Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -226,8 +226,7 @@ def find_indicators_to_limit_loop(indicator_query: str, limit: int, total_fetche
                     for ioc in fetched_iocs)
         last_found_len = len(fetched_iocs)
         total_fetched += last_found_len
-        next_page += 1
-    return iocs, next_page
+    return iocs
```

Comment:
this will probably make some unit tests fail, but other than that, no problem with removing it. Feel free to reconsider whether those unit tests are required anymore - and remove them if you feel they're not.